R/utils.r

```diff
@@ -125,7 +125,7 @@ emacs_version <- function() {
 }
 
 inside_emacs <- function() {
-  Sys.getenv("EMACS") != ""
+  Sys.getenv("INSIDE_EMACS") != ""
```

Comment:
This is not good, because it breaks the systems that use `EMACS`. You need to check for both of them.